src/settings/persistent_settings_handler.cpp

```diff
@@ -29,10 +29,7 @@ namespace
 {
 std::unique_ptr<mp::WrappedQSettings> persistent_settings(const QString& filename)
 {
-    auto ret = mp::WrappedQSettingsFactory::instance().make_wrapped_qsettings(filename, QSettings::IniFormat);
-    ret->setIniCodec("UTF-8");
```

Comment:
I suppose it's UTF-8 by default now, is it?

Reply:
[yup](https://doc.qt.io/qt-6.2/qsettings.html#compatibility-with-older-qt-versions)

Reply:
Cool, thx!